lms/djangoapps/discussion/rest_api/tests/utils.py

```diff
@@ -358,7 +358,7 @@ def assert_query_params_equal(self, httpretty_request, expected_params):
         """
         actual_params = dict(httpretty_request.querystring)
         actual_params.pop("request_id")  # request_id is random
-        assert actual_params == expected_params
+        assert actual_params == expected_params, f"""[\n\t{actual_params} \n\t{expected_params}\n]"""
```

Comment:
What is happening here?

Reply:
Ah! Sorry about this. I use this as a way to see the diff in case of a test failure. Will remove.